lib/workers/repository/finalise/repository-statistics.spec.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  RenovateConfig,
+  getConfig,
+  loadJsonFixture,
+  mockedFunction,
+} from '../../../../test/util';
+import { logger } from '../../../logger';
+import { getPrCache } from '../../../modules/platform/github/pr';
+import { runRenovateRepoStats } from './repository-statistics';
+
+jest.mock('../../../modules/platform/github/pr');
+jest.mock('../../../util/http/github');
+
+const prCache = loadJsonFixture('./pr-cache.json');
```

Comment:
please use fixtures class

Reply:
this is an renovate specific util function, are they all redundant? as i was also requested to not use loadFixture but rather use Fixture class instead